src/components/navBar.jsx

```diff
@@ -0,0 +1,12 @@
+import CartNavItem from './CartNavItem';
+import '../styles/navBar.css';
+
+const NavBar = (props) => {
+  return (
+    <div className="navBarWrapper">
+      <CartNavItem cart={props.cart}/>
+    </div>
+  )
+}
+
+export default NavBar;
```

Comment:
I think if you rename this to something else and then name it back with the capitalized 1st letter it will fix itself.